specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-05-01/containerregistry.json

```diff
@@ -1588,6 +1589,7 @@
       "properties": {
         "type": {
           "description": "The type of trust policy.",
+          "default": "Notary",
```

Comment:
this will cause customer breaking potentially, if default isn't `Notray` before at service side.

Reply:
It was defaulted to `Notary` in service since start. We just didn't expose the value in swagger.